conf/script/prov-m0-reset

```diff
@@ -59,10 +68,10 @@ EOF
 }
 
 TEMP=$(getopt --options h,i: \
-              --longoptions help,ip1:,ip2: \
+              --longoptions help,ip1:,ip2:,interface:,left-iface:,right-iface: \
```

Comment:
`interface` parameter is not documented, but processed.

Reply:
Not critical anymore as we are going to use two parameters for possible different interface names cases. We just can keep the old parameter for some time just for the sake of compatibility with Provisioner and just drop it later completely.